test/unit/active_model/serilizable_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+module ActiveModel
+  class SerializableTest
+    class InstrumentationTest < Minitest::Test
+      def setup
+        @events = []
+
+        ActiveSupport::Notifications.subscribe("!serialize.active_model_serializers") do |name, start, finish, id, payload|
```

Comment:
Better:

`@subscriber = ActiveSupport::Notifications.subscribe(.....` in the setup

and

`ActiveSupport::Notifications.unsubscribe(@subscriber) if defined?(@subscriber)` in the teardown.  less global.